Zend/zend.c

```diff
@@ -1853,7 +1855,11 @@ ZEND_API ZEND_COLD void zend_user_exception_handler(void) /* {{{ */
 		EG(exception) = old_exception;
 	}
 
-	zval_ptr_dtor(&orig_user_exception_handler);
+	if (Z_TYPE(EG(user_exception_handler)) == IS_UNDEF) {
+		zval *tmp = zend_stack_top(&EG(user_exception_handlers));
```

Comment:
Can ``EG(user_exception_handlers)`` be empty?

Reply:
Yes. I missed this somehow.